src/core/Exception.pm

```diff
@@ -1898,11 +1898,13 @@ my class X::Package::UseLib does X::Comp {
 }
 my class X::Package::Stubbed does X::Comp {
     has @.packages;
-    # TODO: suppress display of line number
     method message() {
         "The following packages were stubbed but not defined:\n    "
         ~ @.packages.join("\n    ");
     }
+    multi method gist(::?CLASS:D: :$sorry = True, :$expect = True) {
```

Comment:
Why are these named parameters being given default values but then never used?

Reply:
@jnthn Because I override one defined in role [X::Comb](https://github.com/Gnouc/rakudo/blob/edac1d687099794ac163549adff1863839994653/src/core/Exception.pm#L644)